### DISCUSSION TITLE
Automation sample - receive notification when watch sends message to you

### DISCUSSION THREAD

**Discussion by DIYtechie** [Ludy87/xplora_watch#354] [Q&A]
I can read the messages via xplora_watch.read_message, but can't figure out how to receive a notification when there's a new message for me.

Another thing: does the read message service affect the watch's battery life like the tracking service?

**Comment by Ludy87** [comment]
> Another thing: does the read message service affect the watch's battery life like the tracking service?

Yes and No, if the account is logged out of the server, the integration must be logged in again, which will trigger the location trigger. Simply retrieving the messages will not drain the battery; this can also happen when the watch is switched off.

**Reply by DIYtechie** [reply]
That sounds good. How come the device_tacker is registered as updated every time the read message service is triggered (without the location actually updating)?

**Reply by Ludy87** [reply]
Because some things are combined see: https://github.com/Ludy87/xplora_watch/blob/3d5c0a9de60db9ac5fdfec02dcfa1df75a6ce0ae/custom_components/xplora_watch/device_tracker.py#L156

**Reply by DIYtechie** [reply]
Ok. Good to know. As long as it does not affect the battery that the messages are updates every 30 seconds, that is fine by me.

**Reply by Ludy87** [reply]
I can't promise you that, of course you have to test it yourself, xplora doesn't let you know what's going on.

**Reply by DIYtechie** [reply]
Fair enough. Not expecting any warranties, but from your explanation is sounds like updating messages should not generally try contact the watch but only the server unless I've logged out in the meantime. Good enough. How do you use this integration yourself? Do you receive a notification in Home Assistant, when your kid sends you a message?

**Reply by Ludy87** [reply]
We are not using the watch at the moment because the battery is defective.
